server/metrics/metrics.go

```diff
@@ -1439,6 +1439,24 @@ var (
 		CompressionType,
 	})
 
+	BadCompressionRatioBufferSize = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: bbNamespace,
+		Subsystem: "compressor",
+		Name:      "bad_compression_ratio_buffer_size",
+		Help:      "The size of buffers that increased in size when decompressed",
+	}, []string{
+		CompressionType,
+	})
+
+	CompressionRatio = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: bbNamespace,
+		Subsystem: "compressor",
+		Name:      "compression_ratio",
+		Help:      "The data compression ratio (decompressed bytes / compressed bytes)",
```

Comment:
Since this is only measuring the data compression ratio for pebble and is measuring the ratio for each chunk, maybe "pebble" makes more sense as the subsystem? and also maybe clarify in the Help text that it's per written chunk

Reply:
Done